src/components/index.js

```diff
@@ -7,6 +7,8 @@ import Nav from "./Nav/Nav";
 import VideoCard from "./VideoCard/VideoCard";
 import Spinner from "./Spinner/Spinner";
 import Chips from "./Chips/Chips";
+import Input from "./Input/Input";
+import RequiresAuth from "./RequiresAuth/RequiresAuth";
```

Comment:
You can directly export the components from here.  You can avoid importing here and simply  use export

Reply:
I will try to look. if we can use it for the default export component. thanks for suggestion